frontend/src/views/OrderListView.vue

```diff
@@ -35,13 +39,15 @@ let value = ref();
               {{ order.name }}
             </div>
             <div class="flex flex-col gap-y-2">
-              <Button
-                label="Bearbeiten"
-                icon="pi pi-pencil"
-                severity="secondary"
-                outlined
-                size="small"
-              />
+              <router-link :to="`${ROUTES.ORDER.path}/:id/edit`">
```

Comment:
We need to replace the ":id" part, no?

```suggestion
              <router-link :to="`${ROUTES.ORDER.path}/{order.id}/edit`">
```